backend/routers/community.py

```
@@ -0,0 +1,104 @@
+from fastapi import APIRouter, Depends, HTTPException, status, Request
```

Ruff (0.12.2)

`fastapi.HTTPException` imported but unused

Remove unused import

(F401)